eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       vmImage: "windows-2019"
     steps:
       - checkout: self
-        path: AzureSDKEngSystem/Net/SourceDirectory
+        path: AzureSDKEngrSystem/DotNet/SourcesDirectory
```

Comment:
Did you intend for there to be an "r" in AzureSDKEngrSystem, to try and make the length match? I ask because I fully expect someone will come and think that is a mistake at a later point and try to fix it. If you want this to be a particular length you might consider self documenting it via the name something like: "AzureSDKSourceDirWith49CharLengthToTestLongPaths"

Reply:
Ok, will do that.